test/mailers/user_mailer_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+
+class UserMailerTest < ActionMailer::TestCase
+  test "password_changed" do
```

Comment:
kind of a minor thing, but can you do me a solid and follow the style of the other tests? e.g. `it 'should do a thing' do`.